### ISSUE TITLE
Fix Wallaby test errors

### ISSUE THREAD

**Issue by axelclark** [axelclark/ex338#438]
```bash
  1) test admin creates fantasy team (Ex338Web.AdminCreatesFantasyTeamTest)
     test/integration/admin_creates_fantasy_team_test.exs:5
     ** (Wallaby.JSError) There was an uncaught javascript error:
     
     TypeError: undefined is not a constructor (evaluating 'losingTeamSelects.forEach(function (item, index) {
       item.onchange = filterPlayers;
     })')
       (anonymous function) (http://localhost:4001/js/app.js:10717)
       initModule (http://localhost:4001/js/app.js:42)
       require (http://localhost:4001/js/app.js:59)
       expanded (http://localhost:4001/js/app.js:34)
       (anonymous function) (http://localhost:4001/js/app.js:10661)
       initModule (http://localhost:4001/js/app.js:42)
       require (http://localhost:4001/js/app.js:59)
     
     code: |> visit("/admin")
     stacktrace:
       (wallaby) lib/wallaby/phantom/logger.ex:7: Wallaby.Phantom.Logger.parse_log/1
       (elixir) lib/enum.ex:737: Enum."-each/2-lists^foreach/1-0-"/2
       (elixir) lib/enum.ex:737: Enum.each/2
       (wallaby) lib/wallaby/driver/log_checker.ex:12: Wallaby.Driver.LogChecker.check_logs!/2
       (wallaby) lib/wallaby/browser.ex:709: Wallaby.Browser.visit/2
       test/integration/admin_creates_fantasy_team_test.exs:10: (test)



  2) test user not authorized to visit admin (Ex338.UserNotAuthorizedToAccessAdminTest)
     test/integration/user_not_authorized_to_access_admin_test.exs:5
     ** (Wallaby.JSError) There was an uncaught javascript error:
     
     TypeError: undefined is not a constructor (evaluating 'losingTeamSelects.forEach(function (item, index) {
       item.onchange = filterPlayers;
     })')
       (anonymous function) (http://localhost:4001/js/app.js:10717)
       initModule (http://localhost:4001/js/app.js:42)
       require (http://localhost:4001/js/app.js:59)
       expanded (http://localhost:4001/js/app.js:34)
       (anonymous function) (http://localhost:4001/js/app.js:10661)
       initModule (http://localhost:4001/js/app.js:42)
       require (http://localhost:4001/js/app.js:59)
     
     code: |> visit("/admin")
     stacktrace:
       (wallaby) lib/wallaby/phantom/logger.ex:7: Wallaby.Phantom.Logger.parse_log/1
       (elixir) lib/enum.ex:737: Enum."-each/2-lists^foreach/1-0-"/2
       (elixir) lib/enum.ex:737: Enum.each/2
       (wallaby) lib/wallaby/driver/log_checker.ex:12: Wallaby.Driver.LogChecker.check_logs!/2
       (wallaby) lib/wallaby/browser.ex:709: Wallaby.Browser.visit/2
       test/integration/user_not_authorized_to_access_admin_test.exs:9: (test)



  3) test admin updatest fantasy team (Ex338.AdminUpdatesFantasyPlayerTest)
     test/integration/admin_updates_fantasy_player_test.exs:5
     ** (Wallaby.JSError) There was an uncaught javascript error:
     
     TypeError: undefined is not a constructor (evaluating 'losingTeamSelects.forEach(function (item, index) {
       item.onchange = filterPlayers;
     })')
       (anonymous function) (http://localhost:4001/js/app.js:10717)
       initModule (http://localhost:4001/js/app.js:42)
       require (http://localhost:4001/js/app.js:59)
       expanded (http://localhost:4001/js/app.js:34)
       (anonymous function) (http://localhost:4001/js/app.js:10661)
       initModule (http://localhost:4001/js/app.js:42)
       require (http://localhost:4001/js/app.js:59)
     
     code: |> visit("/admin")
     stacktrace:
       (wallaby) lib/wallaby/phantom/logger.ex:7: Wallaby.Phantom.Logger.parse_log/1
       (elixir) lib/enum.ex:737: Enum."-each/2-lists^foreach/1-0-"/2
       (elixir) lib/enum.ex:737: Enum.each/2
       (wallaby) lib/wallaby/driver/log_checker.ex:12: Wallaby.Driver.LogChecker.check_logs!/2
       (wallaby) lib/wallaby/browser.ex:709: Wallaby.Browser.visit/2
       test/integration/admin_updates_fantasy_player_test.exs:16: (test)

................................................

  4) test update/2 updates a fantasy team's draft queues and redirects (Ex338Web.FantasyTeamControllerTest)
     test/ex338_web/controllers/fantasy_team_controller_test.exs:286
     Assertion with == failed
     code:  assert q1.order() == 2
     left:  3
     right: 2
     stacktrace:
       test/ex338_web/controllers/fantasy_team_controller_test.exs:307: (test)
```

**Comment by axelclark** [comment]
Maybe just delete these tests for now?  Also, if these tests are deleted, need to adjust the test alias.

**Comment by axelclark** [comment]
Closed by #518
